netutils/__init__.py

```diff
@@ -1,3 +1,3 @@
 """Initialization file for library."""
 
-__version__ = "0.2.2"
+__version__ = "0.2.3"
```

Comment:
Thanks for pointing this out, this actually has to be fixed in a few places. You can remove this and squash, i'll update in another PR